omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -182,8 +183,11 @@ private void initSwagger() {
 		        .securityDefinition("basic_auth", new BasicAuthDefinition())
 		        .security(new SecurityRequirement().requirement("basic_auth"))
 		        .consumes("application/json")
-		        .produces("application/json");
-		
+		        .produces("application/json")
+		        .externalDocs(new ExternalDocs()
+				        .description("Find more info on REST Module Wiki")
+				        .url("https://wiki.openmrs.org/display/docs/REST+Module"));
```

Comment:
This should probably use the permanent link to the wiki page: https://wiki.openmrs.org/x/xoAaAQ

The title-based URL could break if the page is moved or renamed in the future.

Reply:
Do you want me to submit a new PR with changes?

Reply:
Yes go ahead and do so.